onnxruntime/core/session/standalone_op_invoker.cc

```diff
@@ -7,6 +7,10 @@
 #include "core/session/ort_apis.h"
 #include <unordered_map>
 
+#if defined(_MSC_VER) && !defined(__clang__)
+#pragma warning(disable : 26400)
```

Comment:
would be good to add a comment for why this is being disabled, and if possible limit the scope where it is disabled.

Reply:
Done.